packages/bundler/src/Bundler.ts

```diff
@@ -105,27 +133,83 @@ export class Bundler implements IBundler {
       userOpHash: sendUserOperationResponse.result,
       wait: (confirmations?: number): Promise<UserOpReceipt> => {
```

Comment:
Give a default value of confirmations = 0 in the method

Reply:
the logic is already like this, so if confirmations is not defined or 0 it will just resolve and send back receipt
```ts
if (confirmations) {
  const latestBlock = await provider.getBlockNumber();
  const confirmedBlocks = latestBlock - userOpResponse.receipt.blockNumber;
  if (confirmations >= confirmedBlocks) {
    clearInterval(intervalId);
    resolve(userOpResponse);
  }
}

clearInterval(intervalId);
resolve(userOpResponse);
```